lib/sycamore/sycamore/tests/unit/transforms/test_aryn_partitioner.py

```diff
@@ -0,0 +1,27 @@
+from sycamore.transforms.aryn_partitioner import ArynPDFPartitioner
+from sycamore.tests.config import TEST_DIR
+import json
+
+
+class MockResponse:
+    def __init__(self) -> None:
+        self.status_code = 200
+
+    def json(self) -> dict:
+        path = TEST_DIR / "resources/data/json/model_server_output_transformer.json"
+        return json.loads(open(str(path), "r").read())
+
+    def text(self) -> str:
+        return ""
+
+
+class TestArynPDFPartitioner:
+    def test_partition(self, mocker) -> None:
+        mocker.patch("requests.post", return_value=MockResponse())
+        with open(TEST_DIR / "resources/data/pdfs/Ray.pdf", "rb") as pdf:
+            ArynPDFPartitioner.partition_pdf(pdf, aryn_token="")
```

Comment:
Let's assert on the response from  ArynPDFPartitioner.partition_pdf call! Same below